src/lib/orbitdb-api.js

```diff
@@ -66,7 +66,14 @@ class OrbitdbAPI extends Express {
         var db_put = asyncMiddleware( async (req, res, next) => {
             let db, hash
             db = await dbm.get(req.params.dbname)
-            hash = await db.put(req.body)
+
+            if (db.type == 'keyvalue') {
+                let params = req.body;
+                hash = await db.put(params.key, JSON.parse(params.value))
```

Comment:
`JSON.parse` is redundant here, the json parser instruction on L14 means that the whole `params` var is already an unpacked object and no longer a string to be parsed.